tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -940,16 +940,16 @@ def eltwise_floor_div(
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
     t1 = setup_tt_tensor(y, device, layout[1], input_mem_config[1], dtype[1])
-    t2 = ttl.tensor.floor_div(t0, t1, output_mem_config=output_mem_config)
+    t2 = ttnn.binary_fmod(t0, t1, memory_config=output_mem_config)
```

Comment:
Why is this not just ttnn.fmod ?

Reply:
ttnn.fmod is a unary op as of now. However, @mouliraj-mcw has overloaded fmod and remainder in his PR [(Link) ](https://github.com/tenstorrent/tt-metal/pull/10838)
I will be making the corresponding changes to binary fmod and remainder once the overload PR is merged.